src/app/zeko/compile_simple/fake/compile_simple.ml

```diff
@@ -27,6 +31,14 @@ module Verification_key = struct
   let hash (Vk x) = x
 
   let hash_var (Var x) = x
+
+  let of_pickles x =
```

Comment:
Pretty sure this isn't correct. You should have another constructor for the real case, otherwise verification of fake proofs will fail.